src/bin/sol-fbp-generator/main.c

```diff
@@ -645,6 +652,7 @@ main(int argc, char *argv[])
     struct sol_fbp_graph graph = {};
     struct type_store *store;
     struct type_description **descs;
+    char temp_file[] = ".sol-generated.c.tmp-XXXXXX";
```

Comment:
There's no reason at all, do you think it would be better if they're not hidden?